src/SFA.DAS.EmployerCommitmentsV2.Web/Controllers/ApprenticeController.cs

```diff
@@ -495,12 +497,34 @@ public async Task<IActionResult> ResumeApprenticeship(ResumeRequestViewModel vie
         public async Task<IActionResult> ApprenticeshipDetails(ApprenticeshipDetailsRequest request)
         {
             var viewModel = await _modelMapper.Map<ApprenticeshipDetailsRequestViewModel>(request);
-            if (viewModel.ApprenticeshipStatus == ApprenticeshipStatus.Stopped)
-            {
-                TempData.AddFlashMessage(ApprenticeStoppedMessage, ITempDataDictionaryExtensions.FlashMessageLevel.Success);
-            }
+
+            if (!TempData.ContainsKey(FlashMessageTempDataKey) &&  viewModel.ApprenticeshipStatus == ApprenticeshipStatus.Stopped)

```

Comment:
do we always show the flash message, when an apprenticeship is stopped?

Reply:
Yes : we have done as part of AC3 from https://skillsfundingagency.atlassian.net/browse/CON-3172